src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/SurveySvcClient.java

```diff
@@ -17,8 +60,37 @@
    * @return List of SurveyClassifierDTO classifier selectors.
    * @throws RestClientException something went wrong making http call.
    */
-  List<SurveyClassifierDTO> requestClassifierTypeSelectors(UUID surveyId)
-      throws RestClientException;
+  @Retryable(
```

Comment:
The `@Retryable` annotation should be used very sparingly. We can't have users waiting 25 seconds for a response.